CONTRIBUTING.md

```diff
@@ -53,18 +71,33 @@ $ cargo test --test <test_name> --features debug
 2. Push your changes back to your fork (`git push origin $your-branch`)
 3. Create a pull request against `master`! (You can also create the pull request first, and we'll merge when ready. This a good way to discuss proposed changes.)
 
-### Goals
+## Conditions for fulfilling a bounty:
 
-There are a few goals of `clap` that I'd like to maintain throughout contributions. If your proposed changes break, or go against any of these goals we'll discuss the changes further before merging (but will *not* be ignored, all contributes are welcome!). These are by no means hard-and-fast rules, as I'm no expert and break them myself from time to time (even if by mistake or ignorance :P).
+1. You should make a pull request which fixes the issue the bounty was promised for
+2. The pull request should be merged by one of the maintainers
 
-* Remain backwards compatible when possible
-  - If backwards compatibility *must* be broken, use deprecation warnings if at all possible before removing legacy code
-  - This does not apply for security concerns
-* Parse arguments quickly
-  - Parsing of arguments shouldn't slow down usage of the main program
-  - This is also true of generating help and usage information (although *slightly* less stringent, as the program is about to exit)
-* Try to be cognizant of memory usage
-  - Once parsing is complete, the memory footprint of `clap` should be low since the main program is the star of the show
-* `panic!` on *developer* error
```

Comment:
I think this point is important to keep.

And WDYT about bringing in clap_derive/CONTRIBUTING.md into this file? I am not sure how many people will look for that file in clap_derive.

Reply:
> I think this point is important to keep.

The goals were just moved to the start of the file

> And WDYT about bringing in clap_derive/CONTRIBUTING.md into this file? I am not sure how many people will look for that file in clap_derive.

clap derive links to this file in this PR

Reply:
> The goals were just moved to the start of the file

My bad, I somehow missed it.

> clap derive links to this file in this PR

I am talking linking `clap_derive` from this since I am not confident in the discoverability of that file.

Reply:
I've added links out to section-specific contrib documentation